docs/manual/common/gettingstarted/IDEs.md

```diff
@@ -1,48 +1,52 @@
-# Import a Lagom build in your IDE
+#  Import Lagom projects into an IDE
 
-Here you will learn how to import projects defined in a Lagom build into your preferred IDE.
+To use existing Lagom projects with Eclipse and IntelliJ, see the section that describes your tool:
 
-## Using Maven
+* [Importing Maven projects](#maven)  
+* [Importing sbt projects into Eclipse](#sbt-eclipse)  
+* [Importing sbt projects into IntelliJ](#sbt-intellij)  
 
-If using Maven, use the m2clipse plugin in Eclipse, or the built in IntelliJ Maven support in IntelliJ, to import your project.
 
-## Using sbt
+## <a name="maven"></a>Importing Maven projects
```

Comment:
A side effect of this processing is that headings using a non-alphanumeric (or blankspace) are not properly styled (see http://www.lagomframework.com/documentation/1.2.x/java/ReadSide.html where everytime 'read-side' is part of a heading, the styling is wrong).